src/libsyncengine/syncpal/syncpalworker.cpp

```diff
@@ -198,13 +198,6 @@ void SyncPalWorker::execute() {
                     setExitCause(ExitCause::WorkerExited);
                 }
                 break;
-            } else if (interruptCondition()) {
```

Comment:
What is the consequence of this change? None?

Reply:
This flag was not used anymore, so I removed it everywhere.
It is actually unrelated to this PR and should have been the topic of a dedicate one.